### PR TITLE
[6263] Tasks is reorganized and runSequence added

### PR DESCRIPTION
**Pull request by damianc**
https://bugzilla.stxnext.pl/show_bug.cgi?id=6272

### CODE REVIEW

**Comment by damianc** [comment]
I'd assign `config.paths` to a variable in gulpfile.js to get path-referencing variables shortened.

File: gulpfile.js [hunk 1]
```diff
@@ -38,11 +82,35 @@ gulp.task('styles:minify', function() {
...
+gulp.task('prod', function(cb) {
+    runSequence(
+        ['clean', 'component:install'],
+        'component:move',
+        ['script:prod', 'styles:app', 'styles:minify', 'copy:images'],
+        cb
+    );
+});
+
+gulp.task('default', function(cb) {
```
Comment (Gruszks) [inline gulpfile.js]: gulp.task('default', ['dev'])

File: config.js [hunk 2]
```diff
@@ -4,19 +4,26 @@ exports.paths = {
             'src/js/jquery.min.js',
```
Comment (Gruszks) [inline config.js]: Remove this.

File: config.js [hunk 3]
```diff
@@ -4,19 +4,26 @@ exports.paths = {
...
             'src/js/slider.jquery.js',
             'src/js/app.js',
-            'src/js/**/*.js'
+            'src/js/**/*.js',
+            'src/components/jquery/dist/jquery.js',
```
Comment (Gruszks) [inline config.js]: Add those on the top..

File: config.js [hunk 4]
```diff
@@ -4,19 +4,26 @@ exports.paths = {
...
+            'src/components/bootstrap/dist/js/bootstrap.js'
         ],
         css: [
-            'src/css/**/*.css'
+            'src/css/**/*.css',
+            'src/components/bootstrap/dist/css/bootstrap.css'
```
Comment (Gruszks) [inline config.js]: Move to the top.

**Pull request merged**